app/views/shared/_cohort_overview.html.erb

```diff
@@ -0,0 +1,80 @@
+<%
+  students_path = @organisation ? :students_organisation_cohort_path : :students_cohort_path
```

Comment:
Let's not compute the paths here. Send it as argument to the partial from the calling site.